firestore/src/android/field_value_android.cc

```diff
@@ -411,8 +429,15 @@ Local<Array<Object>> FieldValueInternal::MakeArray(
 
 Env FieldValueInternal::GetEnv() { return FirestoreInternal::GetEnv(); }
 
-Object FieldValueInternal::ToJava(const FieldValue& value) {
-  return value.internal_ ? value.internal_->object_ : Object();
+Local<Object> FieldValueInternal::ToJava() const {
+  Env env;
+  return object_.get(env);
+}
+
+Local<Object> FieldValueInternal::ToJava(const FieldValue& value) {
+  Env env;
+  return value.internal_ ? value.internal_->object_.get(env)
+                         : Local<Object>(Object());
```

Comment:
I think you can omit the `Object()` argument to the `Local<Object>` constructor.